src/network/packets_processor/packets_queue.cpp

```diff
@@ -0,0 +1,49 @@
+#include "packets_queue.hpp"
+
+namespace taraxa::network {
+
+PacketsQueue::PacketsQueue(size_t max_workers_count) : MAX_WORKERS_COUNT(max_workers_count), act_workers_count_(0) {}
+
+bool PacketsQueue::isProcessingEligible() const {
+  if (packets_.empty() || act_workers_count_.load() >= MAX_WORKERS_COUNT) {
```

Comment:
`act_workers_count_.load() >= MAX_WORKERS_COUNT` I do not understand why we have this there